change/@fluentui-react-ef43dea9-d428-4f7f-b997-c5d0a9d9bf6f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Oh also this should be minor since you're adding a prop
```suggestion
  "type": "minor",
```

---

updated!